mindjunkies/dashboard/tests/test_views.py

```
@@ -158,3 +302,54 @@ def test_verification_wait_view(client, user):
+    assert "components/archive.html" in [t.name for t in response.templates]
+
+
+@pytest.mark.django_db
+def test_archive_view_non_teacher(client, user):
+    client.force_login(user)
+
+    url = reverse("archive_content")
+    response = client.get(url)
+
+    assert response.status_code == 302
+    assert response.url == reverse("teacher_verification_form")
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)